src/utils/httpTransport/index.ts

```diff
@@ -0,0 +1,72 @@
+const METHODS = {
+  GET: 'GET',
+  POST: 'POST',
+  PUT: 'PUT',
+  DELETE: 'DELETE',
+};
+
+// Самая простая версия. Реализовать штучку со всеми проверками им предстоит в конце спринта
+// Необязательный метод
+function queryStringify(data) {
+  if (typeof data !== 'object') {
+    throw new Error('Data must be object');
+  }
+
+  // Здесь достаточно и [object Object] для объекта
+  const keys = Object.keys(data);
+  return keys.reduce((result, key, index) => `${result}${key}=${data[key]}${index < keys.length - 1 ? '&' : ''}`, '?');
```

Comment:
Выглядит очень сложно. Думаю, что можно изменить вот так
```suggestion
  return '?' + Object
    .entries(data)
    .map(([key, value]) => `${key}=${value}`)
    .join('&')
```